dags/process_elife_zip_dag.py

```diff
@@ -90,8 +95,38 @@ def prepare_jats_xml_for_libero(**context):
 
     s3 = get_aws_connection('s3')
     response = s3.get_object(Bucket=DESTINATION_BUCKET, Key=xml_path)
-    xml = etree.parse(BytesIO(response['Body'].read()))
-    return etree.tostring(xml)
+    article_xml = etree.parse(BytesIO(response['Body'].read()))
+
+    # get article id
+    article_id = article_xml.xpath(
+        '/article/front/article-meta/article-id[@pub-id-type="publisher-id"]'
+    )[0].text
+
+    # add jats prefix to jats tags
+    for element in article_xml.iter():
+        if not element.prefix:
+            element.tag = '{http://jats.nlm.nih.gov}%s' % element.tag
```

Comment:
I thought the input, being JATS, to have a correctly defined namespace for the JATS elements; probably at the top level, perhaps as a default namespace. However this doesn't seem to be the case?
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE article PUBLIC "-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD v1.1d3 20150301//EN" "JATS-archivearticle1.dtd">
<article xmlns:xlink="http://www.w3.org/1999/xlink" article-type="article-commentary" dtd-version="1.1d3">
<front><journal-meta>
...
```

Reply:
Correct. I thought this was the case as well but it seems that the declaration of a name is not enough for lxml to actually add the namespace to elements.

Reply:
What I meant was I was expecting a `xmlns` attribute in the root element of the input XML. Turns out it's not there, so :man_shrugging: no need to change anything.

Reply:
oh yeah...:man_shrugging: